### PR TITLE
Neilium toronto

### LINKED ISSUE #7: neilium's checklist
- [x] Make a branch
- [ ] Work locally
- [ ] Make a pull request
- [ ] Collaborate
- [ ] Merge

### PR DESCRIPTION
**Pull request by Neilium**
Create a file with a list of restaurants. Closes #7

### CODE REVIEW

File: neilium-toronto.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Toronto, ON
+
+-The Craft :beers:
+-Lakeview
```
Comment (cbbw) [inline neilium-toronto.md]: where's this place located?  what's good here?

**Review by cbbw — COMMENTED** [review]
please review comments

**Review by fvirag — APPROVED** [review]
sample comment

File: neilium-toronto.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Toronto, ON
 
 -The Craft :beers:
--Lakeview
+-Lakeview (Dundas and Ossington. Its diner food. Good and cheap!)
```
Comment (cbbw) [inline neilium-toronto.md]: sounds good!  thanks!

**Review by cbbw — APPROVED** [review]
sounds good, thanks!  :shipit:

**Pull request merged**